ht-12/index.js

```diff
@@ -0,0 +1,77 @@
+function carousel () {
+  const nextButtons = Array.from(document.querySelectorAll(`[data-slide="next"]`));
+  const prevButtons = Array.from(document.querySelectorAll(`[data-slide="prev"]`));
+  let carouselWrapper;
+  let carouselItems;
+  nextButtons.forEach(nextButton => {
```

Comment:
the callback functions look almost identicaly. let's unify these callbacks. you can use data attributes to pass additional info